cypress/integration/random_reviews.spec.js

```diff
@@ -0,0 +1,29 @@
+describe("Random Reviews", () => {
+    beforeEach(() => {
+        cy.setCookie("Other", "other");
+        cy.setCookie("XSRF-TOKEN", "abcde");
+        cy.visit("/random-reviews");
+    });
+
+    it("shows all random reviews", () => {
+        const expected = [
+            ["Lay", "20 %"],
+            ["Review cycle", "3 year(s)"]
+        ];
+
+        cy.get(".hook-layPercentageRow").each(($el, index) => {
+            cy.wrap($el).within(() => {
+                cy.get(".hook-layPercentageKey").contains(expected[index][0]);
+                cy.get(".hook-layPercentageValue").contains(expected[index][1]);
+            });
+        });
+    });
+
+    it("allows me to change the lay percentage", () => {
```

Comment:
These don't test what they describe either. They are asserting on the presence of the button, not that you can use it.

Reply:
done